features/steps/steps.py

```diff
@@ -331,16 +406,25 @@ def step_impl(context, relationship_type, entity):
 
 @given('A file with {field} "{values}"')
 def step_impl(context, field, values):
-    values = list(map(str.lower, map(lambda s: s.strip('"'), values.split(' or '))))
+    values = strip_split(values, strp = '"', splt = ' or ')
     if field == "Model View Definition":
         conditional_lowercase = lambda s: s.lower() if s else None
         applicable = conditional_lowercase(get_mvd(context.model)) in values
     elif field == "Schema Identifier":
         applicable = context.model.schema.lower() in values
     else:
         raise NotImplementedError(f'A file with "{field}" is not implemented')
-
+ 
     context.applicable = getattr(context, 'applicable', True) and applicable
+    
+@given('Its values')
+def step_impl(context):
+    context._push()
+    instances_unpacked = unpack_sequence_of_entities(
+        context.instances)  # '(#23IfcWall..)' to '#23IfcWall'
+    values = [do_try(lambda : inst.get_info(recursive=True, include_identifier=False, ignore='SourceCRS'), None)
```

Comment:
Does this work?

`ignore` is a sequence of attribute names. but you supply a string. Unfortunately this doesn't throw an exception in python because a string is a sequence of characters, but you'd be trying to ignore the attributes S, o, u, ... effectively.

https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/ifcopenshell-python/ifcopenshell/entity_instance.py#L344

Reply:
It should not work not indeed, but it does: 
```
>>> f.by_type("IfcRigidOperation")[0].get_info()['SourceCRS']
#11=IfcGeometricRepresentationContext($,'Model',3,1.E-05,#9,#10)
>>> f.by_type("IfcRigidOperation")[0].get_info(ignore='SourceCRS')['SourceCRS']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'SourceCRS'
```
I'll change it to be in a list regardless to be on the safe side. 

Could it be that it has something to do with the commented out lines (in ifcopenshell -> entity_instance.get_info())
```
def get_info_(inst):
                            # for ty in ignore:
                            #     if inst.is_a(ty):
                            #         return None
```
The only lines where ignore seems to be used is: 
```
if self.wrapped_data.get_attribute_names()[i] in ignore:
                        continue
```
 -> It will pass, even with only a string
```
>>> 'SourceCRS' in 'SourceCRS'
True
```